pallets/stateful-storage/README.md

```diff
@@ -0,0 +1,61 @@
+# Stateful Storage Pallet
+
+The Stateful Storage pallet provides per-MSA functionality for reading and writing stateful data (where the relevant piece is solely the latest state).
+
+## Summary
+
+When a Schema uses `Paginated` or `Itemized`, the payload data is stored in this pallet.
+Data stored is user-centric instead of time-centric as with the Messages Pallet.
+The pallet storage uses [child storage](https://paritytech.github.io/polkadot-sdk/master/frame_support/storage/child/index.html) making direct query access complex.
+Custom RPCs are provided for easy access to data.
+
+### Paginated Data (`PayloadLocation:Paginated`)
+
+Data is stored in multiple pages of `1_024` bytes in size, each containing a single item of the associated schema.
```

Comment:
Should we reference the pallet constant here in addition to the value `1_024`?

Reply:
If this were only for the Rust docs, I might, but I hesitate with wanting to pull them into the docs.frequency.xyz

Thoughts?

Reply:
We reference an enum above, `PayloadLocation:Paginated`. Might be nice to at least include the name of the constant, makes searching easier for doc updates if/when the constant value changes.

Reply:
@JoeCap08055 updated, see what you think.